spring-cloud-dataflow-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -206,6 +206,10 @@ upgrading from version `1.0.x` to `1.1.x`.  Migration scripts for specific
 database types can be found
 https://github.com/spring-cloud/spring-cloud-task/tree/master/spring-cloud-task-core/src/main/resources/org/springframework/cloud/task/migration[here].
 
+NOTE: If you wish to use an external H2 database instance instead of the one
```

Comment:
Is setting this property to false required to use *any* external db or only an external H2?